examples/gateways/artifact.yaml

```diff
@@ -9,12 +9,13 @@ spec:
   deploySpec:
     containers:
     - name: "artifact-events"
-      image: "argoproj/artifact-gateway"
+      image: "argoproj/artifact-gateway:v0.6"
       imagePullPolicy: "Always"
       command: ["/bin/artifact-gateway"]
     serviceAccountName: "argo-events-sa"
   configMap: "artifact-gateway-configmap"
   eventVersion: "1.0"
+  imageVersion: "v0.6"
```

Comment:
why need this version here as well as above?

Reply:
Its for the internal gateway-transformer image version. It will be removed in v0.7

Reply:
ah okay so this is just to allow v0.6 to work.